index.js

```diff
@@ -4,6 +4,9 @@ if (process.platform === 'win32') {
   spawn = require('cross-spawn')
 }
 
+function noop() {
```

Comment:
Why empty?

Reply:
Because it's a no-op function. It avoids to redefine an empty function every time I need one. (in this PR it's used only once, but there are other places in the file).